openqabot/types/baseconf.py

```diff
@@ -27,8 +27,10 @@ def __call__(
     def normalize_repos(config):
         pass
 
-    def filter_embargoed(self) -> bool:
-        return any(k.startswith("PUBLIC") for k in self.settings.keys())
+    def filter_embargoed(self, flavor: str) -> bool:
+        return any(k.startswith("PUBLIC") for k in self.settings) or any(
```

Comment:
I think here it is safe to have `startswith` but `PUBLIC` can be extended to `PUBLIC_CLOUD`